datasets/wikiproject_taxonomy.halfak_20191202.yaml

```diff
@@ -27,13 +27,7 @@ Culture:
      - WikiProject African diaspora
      - WikiProject Asian Americans
      - WikiProject Assyria
-     - WikiProject Azerbaijan
-     - WikiProject Basque
-     - WikiProject Berbers
-     - WikiProject Igbo
      - WikiProject Franco-Americans
-     - WikiProject Pashtun
-     - WikiProject Taiwan
```

Comment:
Why are these removed?  No hits?

Reply:
For all of these that I suggested to remove, there seems to be a large number of articles that aren't biographies, which makes me think they probably add more noise than signal and are best left in their respective geography categories. I've been using this spreadsheet (https://docs.google.com/spreadsheets/d/1uXelaSPyR6EGmQoUKkf5rsXuMXBNxxqyHKFPdFEn_tc/edit?usp=sharing) as a guide for what is covered by a given WikiProject (it's a distribution of which topics would be applied to articles in a WikiProject if it was removed from the taxonomy so it captures what other sorts of WikiProjects tag the same articles) as well as the What Links Here for the template. For Igbo, Pashtun, Taiwan, etc. the majority don't seem to be biographies but articles about Igbo/Pashtun/Taiwanese places, culture, and history. Not a hard rule obviously, but felt like they were probably adding more noise than signal to the Biography category and if these were going to be included in Biography, we probably should include a bunch of other ethnicity-oriented wikiprojects. WikiProject Berbers probably has the most biographical focus but even that has a lot of non-biography articles (https://en.wikipedia.org/wiki/Special:WhatLinksHere/Template:WikiProject_Berbers). And actually, I think under the same criteria, I probably would remove WikiProject Assyria as well. Thoughts?

Reply:
That makes sense.  We should document this in the commit comments or somewhere else.  I'm open to ideas for how we can help future people understand the change.

Reply:
good point -- I'll update the commit notes as that seems the most straightforward way. I guess eventually, a release notes with actual versioning of the taxonomy makes the most sense to me?